### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by MESTREGUGABr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
-  </body>
-  </html>
+  <h1>welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (gustavolpfa) [inline index.html]: Could you remove the 'Enjoy your stay' string and make the 'Welcome to Potionfy' string red?

Comment (MESTREGUGABr) [reply index.html]: Sure, can do!

**Review by gustavolpfa — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by gustavolpfa — APPROVED** [review]
Well done, Mestre!

**Pull request merged**